app/components/pipeline/teams/Index/chooser.js

```diff
@@ -192,16 +189,6 @@ export default Relay.createContainer(Chooser, {
             }
           }
         }
-        teams(first: 500) @include (if: $isMounted) {
```

Comment:
I think this data may have been left over from a time where we didn't have your selector technology @ticky - can you think of why we need it?